integration-tests/common/common.go

```diff
@@ -384,10 +384,24 @@ func BuildNodeContractPairID(node *client.ChainlinkClient, ocr2Addr string) (str
 }
 
 func (c *Common) Default(t *testing.T, namespacePrefix string) (*Common, error) {
+	productName := "data-feedsv2.0"
```

Comment:
Is this the right product name?

Reply:
it's OCRv2 test => `"data-feedsv2.0"`, it doesn't matter whether it's running on Solana or Ethereum or some other chain.